### PR TITLE
SPEC-317: Add rationale section

### PR DESCRIPTION
**Pull request by durran**

### CODE REVIEW

**Comment by durran** [comment]
@jyemin Added your notes.

File: source/command-monitoring/command-monitoring.rst [hunk 1]
```diff
@@ -112,6 +112,35 @@ This specification defines the monitoring of inidividual commands and in that re
 an event for each command a bulk write executes. Each of these commands, however, must be linked
 together via the same {{operationId}}.
 
+---------
+Rationale
+---------
+
+The command listener API is responsible only for receiving and handling events sent from the lowest
```
Comment (jyemin) [inline source/command-monitoring/command-monitoring.rst]: This prose only makes sense in the context of FAQ #2.  It doesn't stand on its own.

**Comment by durran** [comment]
Moved.

**Comment by jyemin** [comment]
LGTM.  @behackett look good to you?

**Pull request merged**